packages/patternlab/styleguide/source/_patterns/03-organisms/by-author/help-tip.twig

```diff
@@ -0,0 +1,47 @@
+<div
+  class="ma__help-tip mobile-tray ma__help-tip__text--c-primary"
+  id="{{ helpTip.id }}"
+>
+  <span class="ma__help-tip__label" id="label-{{ helpTip.id }}">
+    {% if helpTip.textBefore %}
+      <span>{{ helpTip.textBefore }}</span>
+    {% endif %}
+    <button
+      class="ma__help-tip__trigger {{ helpTip.expanded ? 'ma__help-tip__trigger--active' : '' }}"
+      id="trigger-{{ helpTip.id }}"
+      aria-expanded="{{ helpTip.expanded ? 'true' : 'false' }}"
+      aria-controls="help-tip-content-{{ helpTip.id }}"
+    >
+      <span>{{ helpTip.textTrigger }}</span>
+      <span class="ma_help-tip__label-a11y ma__visually-hidden">Open tooltip to learn more.</span>
```

Comment:
When the help tip is set up for `aria-describedby`, the button for the tooltip in the container is not interactive with screen readers although the tooltip container content is announced followed by the button label. Screen reader users don't miss the information even the tooltip is collapsed.  But, the visually hidden text becomes not necessary as they don't need to open the tooltip anymore.  Let's remove L.16.  I'm sorry that I asked you to make this change though. 

I believe that keeping the help tip as `aria-describedby` help ensure SR users receive the information with the proper content flow while keyboard and mouse users can click the button to display the tooltip content.

Reply:
Are you suggesting adding `aria-describedby` instead? So it will always announce the helptext using screen readers?